StRoot/StFttClusterMaker/StFttClusterMaker.h

```diff
@@ -45,12 +51,18 @@ class StFttClusterMaker: public StMaker {
     float GetThresholdFor( StFttRawHit * hit ) { return 0.0;}
     bool PassTimeCut( StFttRawHit * hit );
 
+
+
     StEvent*             mEvent;
     StFttCollection*     mFttCollection;
     int                  mRunYear;
     bool                 mDebug;
     StFttDb*             mFttDb;
 
+    int mTimeCutMin  = -999;
+    int mTimeCutMax  =  999;
+    int mTimeCutMode = -1; //default - lookup from DB
+
 
     ClassDef( StFttClusterMaker, 1 )
```

Comment:
This should probably have a version = 0, as the class isn't meant to be persistent. Otherwise we'd be incrementing from version 1 to 2 because you've added new data members. But 0 is better in such a case as this.

Reply:
@genevb - fixed in [aa226e4](https://github.com/star-bnl/star-sw/pull/686/commits/aa226e47c51be7cc84e9244929888f382c26deeb)